redaxo/src/addons/install/lib/api/api_core_update.php

```diff
@@ -30,7 +30,7 @@ public function execute()
             throw new rex_api_exception('The requested core version can not be loaded, maybe it is already installed.');
         }
         $version = $versions[$versionId];
-        if (!rex_string::versionCompare($version['version'], rex::getVersion(), '>')) {
+        if (!rex_version::compare($version['version'], rex::getVersion(), '>')) {
```

Comment:
Ist die Rex-Mindest-Version bereits entsprechend hoch gesetzt im Installer?
Gleiches muss unten für metainfo geprüft werden.

Reply:
Guter punkt. Müssen wir automatisieren https://github.com/redaxo/redaxo/issues/3397

Reply:
@staabm dieser punkt war aber doch noch offen? die min-rex-version dürfte nun nicht mehr passen in den addons.

Reply:
oh du hast recht